### PR TITLE
Integration Test :  External RMI access to WebLogic resource using  cluster NodePort Service 

### PR DESCRIPTION
**Pull request by anpanigr**
ref : https://oracle.github.io/weblogic-kubernetes-operator/faq/external-clients/#kubernetes-nodeports

The use case verifies external RMI client access to WebLogic cluster using the NodePort service instead of LoadBalancer tunneling 

Renamed the  the class ItExternalRmiTunneling  to  ItExternalLbTunneling to well represent the usecase
Modified the setupLoadBalaces to use -v as as ingress release version.

Jenkin Result: 

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/5443/

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItExternalNodePortService.java [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+  public void tearDownAll() {
+    if (System.getenv("SKIP_CLEANUP") == null
+        || (System.getenv("SKIP_CLEANUP") != null
+        && System.getenv("SKIP_CLEANUP").equalsIgnoreCase("false"))) {
+      StringBuffer deployIngress = new StringBuffer("kubectl delete -f ");
+      deployIngress.append(Paths.get(RESULTS_ROOT, "cluster.nodeport.svc.yaml"));
+    }
+  }
+
+  private static void createDomainResource(
```
Comment (marinakog) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItExternalNodePortService.java:361]: could you use method from CommonMiiTestUtils.java ?

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItExternalNodePortService.java [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Wait for managed server services and pods are created in namespace {0}",
+          domainNamespace);
+      checkPodReadyAndServiceExists(managedServerPrefix + i, domainUid, domainNamespace);
+    }
+  }
+
+  /**
+   * The external JMS client sends 300 messages to a Uniform Distributed
+   * Queue using load balancer HTTP url which maps to custom channel on
```
Comment (marinakog) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItExternalNodePortService.java:207]: are you using LB here ?

Comment (anpanigr) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItExternalNodePortService.java:207]: No, the usecase does not use LoadBalancer. Modified the description as follows ..
 ... Queue using Nodeport service http url which maps to custom channel on ...

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItExternalNodePortService.java (hunk 1 above)
Comment (anpanigr) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItExternalNodePortService.java:361]: In the current usecase, I create a domain resource with a pre-defined ConfigMap which add JMS resources to the domain.

**Review by marinakog — APPROVED** [review]

**Review by sankarpn — APPROVED** [review]
LGTM

**Pull request merged**